### PR TITLE
Allow views on measurements without required tags to succeed

### PR DESCRIPTION
**Pull request by johanbrandhorst**
See #659. So far this just adds a failing test, but I intend to submit a fix for this once the correct behaviour has been decided.

### CODE REVIEW

File: exporter/prometheus/prometheus_test.go [hunk 1]
```diff
@@ -433,3 +433,78 @@ tests_foo{method="issue961",service="spanner"} 1
...
+		t.Fatal("metric name and labels being duplicated but must be unique")
+	}
+
+	if strings.Contains(output, "error(s) occurred") {
+		t.Fatal("error reported by prometheus registry")
+	}
+
+	want := `# HELP tests_foo foo
+# TYPE tests_foo counter
+tests_foo{key=""} 1
```
Comment (johanbrandhorst) [inline exporter/prometheus/prometheus_test.go]: This is assuming that we would just create an empty value label for missing tags.

**Comment by johanbrandhorst** [comment]
Label this as `do not merge` please (even though CI should fail).

**Comment by bogdandrutu** [comment]
Working on a fix.

**Comment by bogdandrutu** [comment]
Fixed in https://github.com/census-instrumentation/opencensus-go/pull/989

**Pull request closed without merging**